apps/explorer/src/pages/validators/Validators.tsx

```diff
@@ -81,6 +91,13 @@ function ValidatorPageResult(): JSX.Element {
     const lastEpochRewardOnAllValidators =
         epochData?.data[0].endOfEpochInfo?.totalStakeRewardsDistributed;
 
+    const stakingRatio = useMemo(() => {
+        if (!totalSupplyData?.value) return '--';
+        const totalSupplyValue = Number(totalSupplyData?.value);
+        const ratio = (totalStaked / totalSupplyValue) * 100;
+        return formatPercentageDisplay(Number(ratio.toFixed(2)), '--');
+    }, [totalSupplyData, totalStaked]);
+
```

Comment:
works goo just maybe we could avoid using the hardcoded -- because that is returned by the `formatPercentageDisplay` as default  value if passed value is null.
```
const stakingRatio = useMemo(() => {
        let ratio = null;
        if (totalSupplyData?.value && totalStaked) {
            const totalSupplyValue = Number(totalSupplyData.value);
            ratio = Number(((totalStaked / totalSupplyValue) * 100).toFixed(2));
        }
        return formatPercentageDisplay(ratio);
    }, [totalSupplyData, totalStaked]);
```